pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 79
```

Comment:
I'd like to remove this restriction and change to the 88 black default (or even bump up to 100 (that'll be a fun post-standup)), but for now, lets minimize the change surface and keep the current restriction.

Reply:
+1